specification/src/main/asciidoc/platform/ApplicationProgrammingInterface.adoc

```diff
@@ -1454,12 +1454,12 @@ preferences tree defined by the Preferences API. A future version of
 this specification may define the use of the Preferences API by Jakarta EE
 applications.
 
-=== Enterprise JavaBeans™ (EJB) 3.2 Requirements
+=== Jakarta Enterprise Beans (EJB) 3.2 Requirements
 
-This specification requires that a Jakarta EE
+This specification requires that a  Jakarta EE
 product provide support for _enterprise beans_ as specified in the EJB
-specification. The EJB specification is available at
-_http://jcp.org/en/jsr/summary?id=345_ .
+specification. The Jakarta Enterprise Beans specification is available at
+_https://jakarta.ee/specifications/enterprise-beans/3.2.3_ .
 
 This specification does not impose any
 additional requirements at this time. Note that the EJB specification
```

Comment:
Have to change EJB to "Jakarta Enterprise Beans" or "enterprise beans" (depending on context).  The other alternative (if you have a ton of references to EJB) is to declare that the use of "EJB" refers to "Jakarta Enterprise Beans" for certain sections.  Reference the acronym guideline document for exact wording.